From ba8f93bf7bd9fc33dd36436e6919104e31e213ee Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Tue, 23 May 2017 18:04:06 +0800
Subject: [PATCH 1801/5242] MLK-14966-3 clk: makefile: use ARCH_MXC_ARM64 for
 imx dependency

commit  e192969077ed42a29be36ac21f234ca2cfb58ba0 from
https://source.codeaurora.org/external/imx/linux-imx.git

Use ARCH_MXC_ARM64 instead of ARCH_FSL_IMX8QM for i.MX
ARM64 platforms build dependency.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/clk/Makefile |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/clk/Makefile b/drivers/clk/Makefile
index 9964d10b..53dc92c 100644
--- a/drivers/clk/Makefile
+++ b/drivers/clk/Makefile
@@ -71,7 +71,7 @@ obj-$(CONFIG_H8300)			+= h8300/
 obj-$(CONFIG_ARCH_HISI)			+= hisilicon/
 obj-y					+= imgtec/
 obj-$(CONFIG_ARCH_MXC)			+= imx/
-obj-$(CONFIG_ARCH_FSL_IMX8QM)		+= imx/
+obj-$(CONFIG_ARCH_MXC_ARM64)		+= imx/
 obj-$(CONFIG_MACH_INGENIC)		+= ingenic/
 obj-$(CONFIG_ARCH_KEYSTONE)		+= keystone/
 obj-$(CONFIG_MACH_LOONGSON32)		+= loongson1/
-- 
1.7.9.5

